app/server/src/routes/space/space.controller.js

```diff
@@ -17,8 +19,13 @@ const SpaceController = {
         tags,
         image
       );
-      creator.created_spaces.push(space._id);
-      await creator.save();
+      // {user} created a new space called {space.name}, {date.now-space.createdAt} ago.
+      let activity_body = `${creator.name} ${creator.surname} created a new space called ${space.name}, {timeDiff}.`;
```

Comment:
Giving links to the space and the user in the front-end would be nice, how are they going to link? Using the patterns of the bodies?

Reply:
activity schema has user object, with user id, name, image

Reply:
OK, because all the links use IDs, right? But parsing which part of the body refers to what is hard.

Reply:
No need to do that, I send the ids separately

Reply:
In another comment section, we talked about ; this is about that. Linking a name without matching the body string with a type is hard. Rather than front-end checking for every type of activity, we could send the activity type directly, as you suggested, and let them create the activity body.